### DISCUSSION TITLE
Selection of Loan Offers

### DISCUSSION THREAD

**Discussion by RahulSatanure** [ONDC-Official/ONDC-FIS-Specifications#11] [Q&A]
Once Borrower received multiple loan offers from different lenders on buyer app, Can borrower will have choice for applying ( personal/ unsecured BIZ loan) from multiples lenders on the same time or he can only choose one loan offer by comparing with all offers.

**Comment by Antriksh-Parmar** [comment]
Borrower can choose only one offer.

**Reply by ankitagarwaladobe** [reply]
@Antriksh-Parmar 
IIUC, the borrower can choose only one offer in one flow as the select calls are 1 to 1 between BAP and BPP.

But at the same time a borrower should be be apply with other lenders in a new flow. Am I correct?

**Reply by Antriksh-Parmar** [reply]
Yes, however the subsequent searches may or may not yield offers.

**Reply by SanjanaIL** [reply]
@Antriksh-Parmar By subsequent searches, do you mean when the user comes back to the BAP and tries to create another application? If that is the case, what could lead to no offers given that the user had offers in the previous search?
